### PR TITLE
Vulkan: add support for depth resolves.

### PR DESCRIPTION
**Pull request by prideout**
The DepthResolve test now passes with Vulkan, although DoF still
triggers validation errors, which I'll look into next.

### CODE REVIEW

File: filament/backend/CMakeLists.txt [hunk 1]
```diff
@@ -235,8 +235,8 @@ function(build_vkshader SOURCE TARGET_PATH)
             DEPENDS matc ${SOURCE_PATH}
             COMMENT "Building SPIR-V")
 endfunction()
-build_vkshader("BlitColor.vs" "BlitColorVs.spv")
-build_vkshader("BlitColor.fs" "BlitColorFs.spv")
+build_vkshader("BlitDepth.vs" "BlitDepthVs.spv")
+build_vkshader("BlitDepth.fs" "BlitDepthFs.spv")
```
Comment (romainguy) [inline filament/backend/CMakeLists.txt:239]: Do we smol-v compress shaders built outside of materials?

Comment (prideout) [reply filament/backend/CMakeLists.txt:239]: That's an interesting idea, currently our compileRawShader function does _not_ apply smolv compression...

Comment (prideout) [reply filament/backend/CMakeLists.txt:239]: Let me give this a shot.

Comment (prideout) [reply filament/backend/CMakeLists.txt:239]: @romainguy I implemented your idea and added a commit.

**Review by pixelflinger — APPROVED** [review]

**Pull request merged**